contrib/vmap_extractor/vmapextract/loadlib/loadlib.h

```diff
@@ -19,31 +19,16 @@
 #ifndef LOAD_LIB_H
 #define LOAD_LIB_H
 
-#ifdef WIN32
-typedef __int64            int64;
-typedef __int32            int32;
-typedef __int16            int16;
-typedef __int8             int8;
-typedef unsigned __int64   uint64;
-typedef unsigned __int32   uint32;
-typedef unsigned __int16   uint16;
-typedef unsigned __int8    uint8;
-#else
-#include <stdint.h>
-#ifndef uint64_t
-  #ifndef _WIN32
-    #include <linux/types.h>
-  #endif
-#endif
-typedef int64_t            int64;
-typedef int32_t            int32;
-typedef int16_t            int16;
-typedef int8_t             int8;
-typedef uint64_t           uint64;
-typedef uint32_t           uint32;
-typedef uint16_t           uint16;
-typedef uint8_t            uint8;
-#endif
+#include <cstdint>
```

Comment:
It's a C++ standard library header since C++ 11 so this should be fine for any modern PowerPC compiler. Also, cala's not using PowerPC but an Intel based Mac so no worries there.